service_unavailable_page/web/nginx.conf

```diff
@@ -16,6 +16,8 @@ http {
   keepalive_timeout 30;
   port_in_redirect off; # Ensure that redirects don't include the internal container PORT - 8080
 
+  client_max_body_size 2M;
```

Comment:
is this not specific to the service unavailable page?

Reply:
😭

Reply:
rushed PR... yes